### ISSUE TITLE
Bug in Stabilizer for alternating group

### ISSUE THREAD

**Issue by ChrisJefferson** [gap-system/gap#665]
The following code breaks in master (and 4.8.3), it worked in 4.8.0. It is specifically knowledge of `IsAlternatingGroup` which breaks things. The problem was introduced somewhere in the range 4485002a9..4fca53fbea8, which is a range of commits where @stevelinton added a number of optimisations for symmetric and alternating groups. Unfortunately it's not obvious to me what in that range of commits broke things, or if this is some deeper issue pushed to the surface by those commits.

```
gap> g := Group([ (1,2,3), (2,3,4) ]);
Group([ (1,2,3), (2,3,4) ])
gap> IsAlternatingGroup(g);
true
gap> Stabilizer(g, [ [1,2], [3,4] ], OnSetsSets);
Error, Variable: 'stopat' must have an assigned value in
  p := stopat; at /Users/caj/reps/gap/gap/lib/oprt.gi:1291 called from
OrbitStabilizerAlgorithm( G, false, false, gens, acts, rec(
    pnt := d,
    act := act,
    onlystab := true ) ) at /Users/caj/reps/gap/gap/lib/oprt.gi:2861 called from
orbish( G, pnt, gens, acts, act ) at /Users/caj/reps/gap/gap/lib/oprt.gd:863 called from
CallFuncList( StabilizerFunc, arg ) at /Users/caj/reps/gap/gap/lib/oprt.gi:2829 called from
CallFuncList( Stabilizer, Concatenation( [ SymmetricParentGroup( g ) ], arg )
 ) at /Users/caj/reps/gap/gap/lib/gpprmsya.gi:1018 called from
orbish( G, pnt, gens, acts, act ) at /Users/caj/reps/gap/gap/lib/oprt.gd:863 called from
...  at line 3 of *stdin*
you can 'return;' after assigning a value
```

**Comment by stevelinton** [comment]
Thanks for catching this. It was my error.

**Comment by alex-konovalov** [comment]
I will close this - it's fixed by @hulpke in #667.
